src/commonMain/kotlin/app/cash/turbine/channel.kt

```diff
@@ -149,7 +187,8 @@ public suspend fun <T> ReceiveChannel<T>.skipItems(count: Int) {
         val cause = (event as? Event.Error)?.throwable
         throw TurbineAssertionError("Expected $count items but got $index items and $event", cause)
       }
-      is Event.Item<T> -> { /* Success */ }
+      is Event.Item<T> -> { /* Success */
+      }
```

Comment:
Revert. Did an autoformatter do this? We can change it to like
```kotlin
-> {
  // Success
}
```
or something to prevent it from interfering.